CHANGELOG.rst

```diff
@@ -4,6 +4,17 @@ Ansible Playbooks for Confluent Platform - Release Notes
 
 .. contents:: Topics
 
+v7.1.9
+======
+
+Notable enhancements
+-------------
+
+- Updated confluent cli version to 2.38.1
+- Fixed bugs in discovery to generate inventory file with appropriate security protocols
```

Comment:
discovery code is not present with 7.1. it was introduced in 7.4, please remove this from all versions lower than 7.4

Reply:
Done